coldfront/plugins/qumulo/tests/test_tasks.py

```diff
@@ -245,3 +261,256 @@ def test_conditionally_update_storage_allocation_statuses_checks_all_pending_all
+                    "allocation"
+                ).get(
+                    value=path.rstrip("/"),
+                    allocation_attribute_type=self.storage_filesystem_path_attribute_type,
+                    allocation__status=self.status_active,
+                )
+                allocation = storage_filesystem_path_attribute.allocation
+
+                storage_quota_attribute_type = AllocationAttribute.objects.get(
+                    allocation=allocation,
+                    allocation_attribute_type=self.storage_quota_attribute_type,
+                )
+                allocation_attribute_usage = (
+                    storage_quota_attribute_type.allocationattributeusage
+                )
+            except AllocationAttribute.DoesNotExist:
+                # When the storage_path_attribute for path is not found,
+                # the allocation_attribute_usage should not exist.
+                self.assertIsNone(allocation_attribute_usage)
+                continue
+
+            self.assertEqual(allocation_attribute_usage.value, 0)
+            self.assertEqual(allocation_attribute_usage.history.first().value, 0)
+            self.assertEqual(allocation_attribute_usage.history.count(), 1)
+
+    @patch("coldfront.plugins.qumulo.tasks.QumuloAPI")
+    def test_after_getting_daily_usages_from_qumulo_api(
+        self, qumulo_api_mock: MagicMock
+    ) -> None:
+        qumulo_api = MagicMock()
+        qumulo_api.get_all_quotas_with_usage.return_value = self.mock_quota_response
+        qumulo_api_mock.return_value = qumulo_api
+
+        try:
+            ingest_quotas_with_daily_usage()
+        except:
+            self.fail("Ingest quotas raised exception")
+
+        base_quotas = get_mock_quota_base_allocations(self.STORAGE2_PATH)
+        for path, quoata_data in base_quotas.items():
+            storage_filesystem_path_attribute = AllocationAttribute.objects.select_related(
+                "allocation"
+            ).get(
+                value=path.rstrip("/"),
+                allocation_attribute_type=self.storage_filesystem_path_attribute_type,
+                allocation__status=self.status_active,
+            )
+
+            allocation = storage_filesystem_path_attribute.allocation
+            storage_quota_attribute = AllocationAttribute.objects.get(
+                allocation=allocation,
+                allocation_attribute_type=self.storage_quota_attribute_type,
+            )
+
+            allocation_attribute_usage = (
+                storage_quota_attribute.allocationattributeusage
+            )
+
+            usage = int(quoata_data["usage"])
+            self.assertEqual(allocation_attribute_usage.value, usage)
+            self.assertEqual(allocation_attribute_usage.history.first().value, usage)
+            self.assertGreater(allocation_attribute_usage.history.count(), 1)
+
+    @patch("coldfront.plugins.qumulo.tasks.QumuloAPI")
+    def test_doesnt_ingest_sub_allocation_data(
```

Comment:
@harterjwustl [minor] doesnt --> does_not

Reply:
hhhhmmmmmmmmmmmmmmm.......how strongly do you feel about this?